services/galley/test/integration/API.hs

```diff
@@ -498,8 +495,6 @@ postCryptoMessageVerifyMsgSentAndRejectIfMissingClient = do
       liftIO $ assertBool "unexpected equal clients" (bc /= bc2)
       assertNoMsg wsB2 (wsAssertOtr qconv qalice ac bc cipher)
 
--- @END
-
 -- @SF.Separation @TSFI.RESTfulAPI @S2
 -- This test verifies basic mismatch behavior of the the JSON endpoint.
 postCryptoMessageVerifyRejectMissingClientAndRespondMissingPrekeysJson :: TestM ()
```

Comment:
Forgot to delete `@SF` comment and `@END` comment from this test? Also from `postMessageQualifiedLocalOwningBackendIgnoreMissingClients`?

Reply:
Thanks, fixed!